src/jobs/jobs.controller.ts

```
@@ -370,4 +370,53 @@ export class JobsController {
       user.id,
     );
   }
+
+
+    @Post(':id/extend')
+  async extendJob(
+    @Param('id') id: string,
+    @Body() extendJobDto: ExtendJobDto,
+  ) {
```

ESLint

[error] Replace `⏎····@Param('id')·id:·string,⏎····@Body()·extendJobDto:·ExtendJobDto,⏎··` with `@Param('id')·id:·string,·@Body()·extendJobDto:·ExtendJobDto`

(prettier/prettier)